src/index.ts

```diff
@@ -118,14 +156,25 @@ export class GitHubStrategy<User> extends OAuth2Strategy<
       },
       verify
     );
-    this.scope = scope ?? this.USER_EMAIL_SCOPE;
+    this.scope = this.getScope(scope);
     this.allowSignup = allowSignup ?? true;
     this.userAgent = userAgent ?? "Remix Auth";
   }
 
+  //Allow users the option to pass a scope string, or typed array
+  protected getScope(scope: GitHubStrategyOptions["scope"]) {
```

Comment:
This should be private, I don't think you want to change it in case you extend the strategy
```suggestion
  private getScope(scope: GitHubStrategyOptions["scope"]) {
```

Reply:
Yep you are right! resolved :)